06week/ticTacToe/script.js

```diff
@@ -1,5 +1,54 @@
 'use strict';
 
 document.addEventListener('DOMContentLoaded', () => {
-  // Your code here
+
+  const winState = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    // horizontal wins
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    // vertical wins
+    [0,4,8],
+    [2,4,6]
+    // diagonal wins
+  ];
+  let playerTurn = 'X';
+
+
+  let boxes = document.querySelectorAll("[data-cell]");
+
+
+  boxes.forEach(box => {
+    box.addEventListener("click", thisBox => {
+      if (!thisBox.target.textContent) {
```

Comment:
nice! this looks great, cleaner JS would be to make this into an if/ else if/else if to avoid the nested if statements, however this works good.